services/webhooks2tasks/src/handlers/githubPush.js

```diff
@@ -19,6 +19,17 @@ async function githubPush(webhook: WebhookRequestData, project: Project) {
     const branchName = body.ref.toLowerCase().replace('refs/heads/','')
     const sha = body.after
 
+    // for each item in the body.commits array, search for our regular expression; this is an arroy of boolean
+    // which is .reduced into a single true if any are true.
+    const skip_deploy = body.commits.map( commit => Array.isArray(commit.message.match(/\[skip deploy\]|\[deploy skip\]/i)))
+                                    .reduce( function(a,i,x,z) { return a||i })
+    if (skip_deploy) {
+      sendToLagoonLogs('info', project.name, uuid, `${webhooktype}:${event}:skipped`, meta,
```

Comment:
this now needs an additional entry in logs2slack here: https://github.com/amazeeio/lagoon/blob/master/services/logs2slack/src/readFromRabbitMQ.js#L57
or we would never receive slack notifications for it

Reply:
Those are in: 7e0f8fc57 d91a3af9c 318b3f02d